serverlessworkflow/modules/ROOT/pages/eventing/handling-events-on-workflows.adoc

```diff
@@ -122,29 +120,25 @@ If you want the same action to be executed when any of the event types are recei
   }
 ----
 
+You can also remove the `exclusive` property, as `true` is the default value.
 
-== Starting a workflow instance
-
-An event state can be used to start a new workflow instance. To enable this functionality, the `start` property in the workflow header should be set to the event state name. The workflow instance will be started when there is an xref:eventMatch[event match]. 
-
-Given the previously defined `waitForEvent` state, you set as starting event by adding following code at workflow header. 
+To start a new workflow instance, set the `start` property to the event state name in the workflow header. The workflow instance starts when there is an event match. Using the previous example of event state definition, add the following code in the workflow header:
 
+.Start a workflow instance
 [source,json]
 ----
 "start":"waitForEvent"
 ----
 
-== Resuming workflow execution
-
-An Event state can be used to pause an existing workflow instance. When the Workflow execution reaches a non starting Event state, it pauses till there is an xref:eventMatch[event match] for that particular workflow instance. 
+An event state can also be used to pause an existing workflow instance. When the workflow execution reaches an event state, which is not starting, then the execution is paused until there is an event match for that workflow instance.
 
-Similar to xref:working-with-callbacks.adoc[Callback state], the instance to be resumed is identified by `kogitoprocrefid` CloudEvent Attribute or calculated according to xref:event-correlation-with-workflows.adoc[Event Correlation] functionality. While callback state is used for _fire&wait_, event state covers _wait&fire_ scenarios.
+Similar to the callback state in Serverless Workflow, the workflow instance to be resumed is identified by `kogitoprocrefid` CloudEvent attribute or calculated according to the event correlation functionality. While callback state is used for _fire&wait_ scenaiors, event state covers _wait&fire_ scenarios. For more information about callback state, see xref:eventing/working-with-callbacks.adoc[Callback state in Serverless Workflow].
```

Comment:
You accidentally remove the link to correlation functionality

Reply:
Done, added the link